docs/source/intro.rst

```diff
@@ -0,0 +1,72 @@
+.. _intro:
+
+Introduction: why is it complex to assess DEM accuracy and precision?
+=====================================================================
+
+Digital Elevation Models are numerical representations of elevation. They are generated from different instruments (e.g., radiometer, radar, lidar), acquired in different conditions (e.g., ground, airborne, satellite), and using different post-processing techniques (e.g., stereophotogrammetry, interferometry).
+
+While some complexities are specific to certain instruments and methods, all DEMs generally have:
+
+- an **arbitrary Ground Sampling Distance (GSD)** that does not necessarily represent their underlying spatial resolution,
+- an **georeferenced positioning subject to shifts, tilts or other deformations** due to inherent instrument errors, noise, or associated post-processing schemes,
+- a **large number of outliers** that can originate from various sources (e.g., photogrammetric blunders, clouds).
+
+These factors lead to difficulties in assessing the accuracy and precision of DEMs, which are necessary to perform further analysis.
+
+In ``xdem``, we provide a framework with state-of-the-art methods published in the scientific literature to make DEM calculations consistent, reproducible, and easy.
+
+Accuracy and precision
+**********************
+
+Both accuracy and precision are important factors to account for when analyzing DEMs:
+
+- the **accuracy** (systematic error) of a DEM describes how close a DEM is to the true location of measured elevations on the Earth's surface,
+- the **precision** (random error) of a DEM describes the typical spread of its error in measurement, independently of a possible bias from the true positioning.
+
+TODO: Add a little schematic!
+
+Absolute or relative accuracy
+*****************************
+
+The measure of accuracy can be further divided into two aspects:
+
+- the **absolute accuracy** of a DEM describes the average shift to the true positioning. Studies interested in analyzing features of a single DEM in relation to other georeferenced data might give great importance to this potential bias.
+- the **relative accuracy** of a DEM is the potential shifts, tilts, and deformations in relation to other elevation data that does not necessarily matches a given referencing. Studies interested in comparing DEMs between themselves might be only interested in this accuracy.
```

Comment:
it's weird to have a singular then plural, so something's wrong. Maybe "the **relative accuracy** of a DEM is related to the potential shifts, tilts, and deformations with reference to other elevation..."